proto/v1alpha1/state.proto

```diff
@@ -166,8 +166,9 @@ message WatchOptions {
     int32 tail_events = 2;
     resource.LabelQuery label_query = 3;
     resource.IDQuery id_query = 4;
+    bool aggregated = 5;
 }
 
 message WatchResponse {
-    Event event = 1;
+    repeated Event event = 1;
```

Comment:
this change is backwards compatible btw, `repeated` of one element is same as a single element.